docs/cloud-messaging/first-message.md

```diff
@@ -73,6 +73,75 @@ which records the number of messages sent and opened on Apple and Android
 devices, along with data for "impressions" (notifications seen by users) for
 Android apps.
 
+## Handling interaction
+
+The default behavior for notifications (e.g. by pressing them) on both Android & iOS is to open the application. If the application is terminated,
```

Comment:
I still don't think this parenthetical is at all required.  Two points:

-- Even setting aside non-sighted usage, "pressing" is nonstandard;  I've heard many references to "tapping" on mobile GUIs, but I strongly prefer the agnostic "selecting" for such cases.  

-- "e.g." is discourage by the Google developer style guide, here: https://developers.google.com/style/abbreviations#dont-use

In any case, I don't think this parenthetical adds anything.  Without it, the sentence seems perfectly clear to me. The only clarification I can think of might be to clarify that this behavior is initiated by the interaction, which I think can be safely assumed.  But if you and Kevin feel it is necessary to clarify, how about "When users select a notification, the default behavior on both . . . ."  ?

Reply:
SGTM.